syntax = "proto3";

package service;

service RaffleService {
  rpc GetRaffleStatus (RaffleRequest) returns (RaffleResponse) {}
}

message RaffleRequest {
  string raffle_id = 1;
}

message RaffleResponse {
  string status = 1;
  string message = 2;
}